course/include/Isometry.hpp

```diff
@@ -91,5 +92,44 @@ inline std::ostream& operator<<(std::ostream& os, const Vector3& vect) {
               << ", z: " << std::to_string(static_cast<int>(vect.z())) << ")";
 }
 
+class Matrix3 {
+   public:
+    Matrix3(const double m00, const double m01, const double m02, const double m10, const double m11, const double m13,
+            const double m21, const double m22, const double m23){
+                values_.push_back(m00);
+                values_.push_back(m01);
+                values_.push_back(m02);
+                values_.push_back(m10);
+                values_.push_back(m11);
+                values_.push_back(m13);
+                values_.push_back(m21);
+                values_.push_back(m22);
+                values_.push_back(m23);
+            }
+    Matrix3(const std::initializer_list<double>& row_1, const std::initializer_list<double>& row_2,
+            const std::initializer_list<double>& row_3){
+                for(auto i : row_1){
```

Comment:
nit: reserve the `i` name for counters. It's so conventional that people expect it to be like that. Call it `item`, or `cell`, or `number` .

Reply:
Done!